vello_tests/tests/known_issues.rs

```diff
@@ -78,6 +78,38 @@ fn many_bins_cpu() {
     many_bins(true);
 }
 
+/// Test for <https://github.com/linebender/vello/issues/1061>
+#[test]
+#[should_panic]
+fn test_layer_size() {
+    let mut scene = Scene::new();
+    scene.fill(
+        vello::peniko::Fill::NonZero,
+        Affine::IDENTITY,
+        vello::peniko::color::AlphaColor::from_rgb8(0, 255, 0),
+        None,
+        &Rect::from_origin_size((0.0, 0.0), (60., 60.)),
+    );
+    scene.push_layer(
+        vello::peniko::Compose::Clear,
+        1.0,
+        vello::kurbo::Affine::IDENTITY,
+        &Rect::from_origin_size((20.0, 20.0), (20., 20.)),
+    );
+    scene.pop_layer();
+    scene.fill(
+        vello::peniko::Fill::NonZero,
+        Affine::IDENTITY,
+        vello::peniko::color::AlphaColor::from_rgb8(255, 0, 0),
+        None,
+        &Rect::from_origin_size((20.0, 20.0), (20., 20.)),
+    );
```

Comment:
Can we maybe make this a hollow shape? That would actually then be testing that the clear compose is actually doing anything. But possibly disregard this if I'm misunderstanding what the clear compose does

Reply:
I seen many WPT test that did exactly like I did and it's really painful to dbg, so in https://github.com/linebender/vello/pull/1062/commits/739e6a18619ffe631d67b3af9ca1ab4efcff7e8d I draw red on top of green, and then expect layer to clear whole red area (we are currently clearing too much).